slam-ws/src/nav_rtab/launch/rtab.launch

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<launch>
+
+    <!-- Visual SLAM -->
+    <!-- Remove "delete_db_on_start" to build on an existing map, saved in ~/.ros/rtabmap.db-->
+    <node name="rtabmap" pkg="rtabmap_ros" type="rtabmap" output="screen" args="--delete_db_on_start">
```

Comment:
Can you follow the format I used for `move_base.launch` and `odom.launch`? That is, add an `<arg>` called `robot_model` and use the `<group>` if statement structure to call rtab. **NOTE:** you don't need to implement rtab for `husky_gazebo`,  just put a comment that it is un-implemented for `husky_gazebo`